.github/workflows/ci.yml

```diff
@@ -8,64 +8,62 @@ jobs:
     environment: TodoApp_Environment
     steps:
     - uses: actions/checkout@v2
-   # - run: echo Hello World
-
-    
-
+  
     -  name: Login to DockerHub
        uses: docker/login-action@v1 
        with:
           username: ${{secrets.DOCKERHUB_USERNAME}}
           password: ${{secrets.DOCKERHUB_TOKEN}}
       
-    
-
     - name: Build Unit Tests
       run: docker build --target test --tag todo-app:test .
-     
 
     - name: Run Unit Test
       run: docker run --env-file .env.test todo-app:test  /app/todo_app/tests
 
-  # - name:  Build End to End tests
-   #   run: docker build --target test --tag todo-app:test .
-
-    - name: Run the end to end test
-     # run: docker run -e TRELLO_BOARD_ID=${{ secrets.TRELLO_BOARD_ID }} -e TRELLO_TOKEN=${{ secrets.TRELLO_TOKEN }} -e TRELLO_API_KEY=${{ secrets.TRELLO_API_KEY }} -e ID_LIST_NEWLIST=${{ secrets.ID_LIST_NEWLIST }} -e ID_LIST_TODO=${{ secrets.ID_LIST_TODO }} -e ID_LIST_DONE=${{ secrets.ID_LIST_DONE }} -e ID_LIST_DOING=${{ secrets.ID_LIST_DOING }} -e CARDS_TRELLODONE_CARD=${{ secrets.CARDS_TRELLODONE_CARD }} -e CARDS_TODO_CARD_TWO=${{ secrets.CARDS_TODO_CARD_TWO }} -e CARDS_TODO_CARD_THREE=${{ secrets.CARDS_TODO_CARD_THREE }} -e CARDS_TODO_CARD_ONE=${{ secrets.CARDS_TODO_CARD_ONE }} -e CARDS_DEVOPSTRELLOBOARD=${{ secrets.CARDS_DEVOPSTRELLOBOARD }} -e FLASK_APP=${{ secrets.FLASK_APP }} -e FLASK_ENV=${{ secrets.FLASK_ENV }} -e SECRET_KEY=${{ secrets.SECRET_KEY }}  -p 5100:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app  todo-app:test /app/todo_app/SeleniumTest
-
-      run: docker run -e FLASK_APP=${{ secrets.FLASK_APP }} -e FLASK_ENV=${{ secrets.FLASK_ENV }} -e SECRET_KEY=${{ secrets.SECRET_KEY }} -e TRELLO_API_KEY=${{ secrets.TRELLO_API_KEY }} -e TRELLO_TOKEN=${{ secrets.TRELLO_TOKEN }} -e TRELLO_BOARD_ID=${{ secrets.TRELLO_BOARD_ID }} -e ID_LIST_TODO=${{ secrets.ID_LIST_TODO }} -e ID_LIST_DONE=${{ secrets.ID_LIST_DONE }} -e ID_LIST_DOING=${{ secrets.ID_LIST_DOING }} -e CARDS_TRELLODONE_CARD=${{ secrets.CARDS_TRELLODONE_CARD }} -p 5100:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app  todo-app:test /app/todo_app/SeleniumTest
-
+  
+   # - name: Run the end to end test
+   #   run: docker run -e FLASK_APP=${{ secrets.FLASK_APP }} -e FLASK_ENV=${{ secrets.FLASK_ENV }} -e SECRET_KEY=${{ secrets.SECRET_KEY }} -e TRELLO_API_KEY=${{ secrets.TRELLO_API_KEY }} -e TRELLO_TOKEN=${{ secrets.TRELLO_TOKEN }} -e TRELLO_BOARD_ID=${{ secrets.TRELLO_BOARD_ID }} -e ID_LIST_TODO=${{ secrets.ID_LIST_TODO }} -e ID_LIST_DONE=${{ secrets.ID_LIST_DONE }} -e ID_LIST_DOING=${{ secrets.ID_LIST_DOING }} -e CARDS_TRELLODONE_CARD=${{ secrets.CARDS_TRELLODONE_CARD }} -p 5100:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app  todo-app:test /app/todo_app/SeleniumTest
```

Comment:
Was this switched off to speed up your deployments? If so we should reintroduce it when we can so that we get the regression protection it's designed to afford us!

Reply:
done